### PR TITLE
Update url argument description

### PR DESCRIPTION
**Pull request by TalGumi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
@TalGumi Hi, please update release notes.

File: Packs/ThreatGrid/Integrations/ThreatGrid/1_8490e145-c880-4802-a995-9ec32e4f82e2 [hunk 1]
```diff
@@ -0,0 +1 @@
+{"api_version":2,"id":1234,"request_id":"req-7","data":{"items":[{"path":"w.dll","sample_id":"123","severity":100,"aid":1}],"items_per_page":1000}}
```
Comment (daryakoval) [inline Packs/ThreatGrid/Integrations/ThreatGrid/1_8490e145-c880-4802-a995-9ec32e4f82e2:1]: What is this file?

File: Packs/ThreatGrid/ReleaseNotes/2_0_3.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Cisco Secure Malware Analytics (Threat Grid) v2
+
+- Add validation to URL argument
```
Comment (daryakoval) [inline Packs/ThreatGrid/ReleaseNotes/2_0_3.md:6]:
```suggestion
- Added a validation to *URL* argument in the ***threat-grid-url-search*** command.
```

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.yml [hunk 3]
```diff
@@ -510,13 +510,13 @@ script:
     arguments:
     - name: url
       required: true
-      description: The URL to search for.
+      description: The URL to search for (please provide the URL in the format http://example.com:80/. note that ThreatGris only support '.com' domains).
```
Comment (daryakoval) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.yml:513]: Please add this to the Integration readme as well, (Packs/ThreatGrid/Integrations/ThreatGridv2/README.md), to the commands argument description.

File: Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.yml [hunk 4]
```diff
@@ -510,13 +510,13 @@ script:
...
       isArray: false
     outputs:
     - contextPath: ThreatGrid.search.url
       description: URL item
       type: String
-    description: Search urls. Please provide the URL in the format http://example.com:80/ .
+    description: Search urls. Please provide the URL in the format http://example.com:80/ (note that ThreatGris only support '.com' domains).
```
Comment (daryakoval) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/ThreatGridv2.yml:519]: Please add this to the Integration readme as well, (Packs/ThreatGrid/Integrations/ThreatGridv2/README.md), to the command description.

File: Packs/ThreatGrid/Integrations/ThreatGrid/1_8490e145-c880-4802-a995-9ec32e4f82e2 (hunk 1 above)
Comment (TalGumi) [reply Packs/ThreatGrid/Integrations/ThreatGrid/1_8490e145-c880-4802-a995-9ec32e4f82e2]: It's from the unit test, I deleted this.

**Review by daryakoval — APPROVED** [review]

File: Packs/ThreatGrid/Integrations/ThreatGridv2/README.md [hunk 5]
```diff
@@ -674,7 +674,7 @@ Search urls. Please provide the URL in the format http://example.com:80/ .
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| url | The URL to search for. | Required | 
+| url | The URL to search for (please provide the URL in the format http://example.com:80/. note that ThreatGris only support '.com' domains). | Required |
```
Comment (meital-aizen) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/README.md:677]: @TalGumi There's a typo in "ThreatGris" + Note with capital letter and not note

File: Packs/ThreatGrid/Integrations/ThreatGridv2/README.md [hunk 6]
```diff
@@ -664,7 +664,7 @@ Returns data regarding the annotations of the analysis
 
 ### threat-grid-url-search
 ***
-Search urls. Please provide the URL in the format http://example.com:80/ .
+Search urls. Please provide the URL in the format http://example.com:80/ (note that ThreatGris only support '.com' domains).
```
Comment (meital-aizen) [inline Packs/ThreatGrid/Integrations/ThreatGridv2/README.md:667]: @TalGumi There's a typo in "ThreatGris" + Note with capital letter and not note

**Pull request merged**